### PR TITLE
US#97121 Write mashup to make editor panel in test steps transparent

### PR DESCRIPTION
**Pull request by kruglik-alexey**

### CODE REVIEW

File: Transparent CK Editor Panel/readme.mkd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Transparent CK Editor Panel
+The Transparent CK Editor Panel mashup make floating CK Editor panel transparent to help see elements behind it.
+
+For example it helps to see previous Test Step while editing another one.
+
+![Transparent CK Editor Panel in Test Steps](https://github.com/kruglik-alexey/TP3MashupLibrary/blob/feature/us97121_transparent_cke_panel/Transparent%20CK%20Editor%20Panel/transaprent.png)
```
Comment (roadhump) [inline Transparent CK Editor Panel/readme.mkd]: set path relative to merged mashup, for example as here https://raw.githubusercontent.com/TargetProcess/TP3MashupLibrary/master/Dark%20Theme/README.mkd

Comment (kruglik-alexey) [inline Transparent CK Editor Panel/readme.mkd]: Will do it after QA

File: Transparent CK Editor Panel/readme.mkd [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Transparent CK Editor Panel
+The Transparent CK Editor Panel mashup make floating CK Editor panel transparent to help see elements behind it.
```
Comment (dkanashevich) [inline Transparent CK Editor Panel/readme.mkd]: make**s**

File: Transparent CK Editor Panel/readme.mkd [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+For example it helps to see previous Test Step while editing another one.
```
Comment (dkanashevich) [inline Transparent CK Editor Panel/readme.mkd]: **the** previous

File: Transparent CK Editor Panel/readme.mkd [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+![Transparent CK Editor Panel in Test Steps](https://raw.githubusercontent.com/kruglik-alexey/TP3MashupLibrary/feature/us97121_transparent_cke_panel/Transparent%20CK%20Editor%20Panel/transaprent.png)
```
Comment (dkanashevich) [inline Transparent CK Editor Panel/readme.mkd]: Don't forget to change image url from your fork to the original repo

**Pull request merged**